requirements.txt

```diff
@@ -15,3 +15,4 @@ python-dotenv==0.14.0
 regex==2020.9.27
 requests==2.24.0
 werkzeug==0.16.1
+python-dateutil==2.8.2
```

Comment:
is this needed 😅  Can you remove this and then I am happy to approve :)

Reply:
Not really but the build was failing before adding it, shall I revert the change?

Reply:
alright, then I guess you can leave it :)